docs/REQUIREMENTS.md

```diff
@@ -666,6 +666,14 @@ INGRESS_NGINX_CONFIG:
 NFS_CLIENT_NAME: nfs-subdir-external-provisioner-sas
 ```
 
+You must explicitly set the value for `V4M_STORAGECLASS` in your ansible-vars.yaml file as shown below to a pre-existing Storage Class (for example: `local-storage`) prior to installing `cluster-logging` or `cluster-monitoring` with [viya4-deployment](https://github.com/sassoftware/viya4-deployment.git).
+
+```yaml
+## Logging and Monitoring
+V4M_STORAGECLASS = local-storage
+```
```

Comment:
I would prefer to have all of the code in an easy snippet that folks could copy and paste into their ansible vars file for the deployment tooling. If needed we should provide guidance to these items after the snippet.

For example the snippet should look like:

```yaml
## 3rd Party
### Ingress Controller
INGRESS_NGINX_CONFIG:
  controller:
    service:
      externalTrafficPolicy: Cluster or Local # Refer to Load Balancer type table above
      # loadBalancerIP: <your static ip> # Assigns a specific IP for your loadBalancer
      loadBalancerSourceRanges: [] # Not supported on open source kubernetes - https://kubernetes.io/docs/reference/kubernetes-api/service-resources/service-v1/
      annotations:

### NFS Subdir External Provisioner - SAS default storage class
# Updates to support open source Kubernetes 
NFS_CLIENT_NAME: nfs-subdir-external-provisioner-sas

## Logging and Monitoring
V4M_STORAGECLASS = local-storage
```
**NOTE**: You must explicitly set the value for `V4M_STORAGECLASS` in your ansible-vars.yaml file as shown below to a pre-existing Storage Class (for example: `local-storage`) prior to installing `cluster-logging` or `cluster-monitoring` with [viya4-deployment](https://github.com/sassoftware/viya4-deployment.git).

This is much cleaner and easier for folks to use and digest.